.env.example

```diff
@@ -267,6 +267,14 @@ QFIELDCLOUD_PASSWORD_LOGIN_IS_ENABLED=1
 # DEFAULT: {}
 SOCIALACCOUNT_PROVIDERS='{}'
 
+# Header name used to specify the Identity Provider (IDP) when authenticating with third-party.
+# DEFAULT: X-QFC-IDP-ID
+QFIELDCLOUD_IDP_ID_HEADER_NAME=X-QFC-IDP-ID
+
+# Header name used to provide the ID token when authenticating with third-party.
+# DEFAULT: X-QFC-ID-Token
+QFIELDCLOUD_ID_TOKEN_HEADER_NAME=X-QFC-ID-Token
+
```

Comment:
Do we actually need them as envvars for now? I would say no, not really.

It would be enough to be in settings.py and eventually expose them to the .env one day, no?

@lukasgraf @gounux ?

Reply:
Yes, I actually agree with this.
It's good to have a mechanism in place for them to be configurable on the server side, and to get clients to implement that mechanism before a broad rollout, but as of now I don't see a need to *actually* configure them to a different value. And it would keep the `.env` simpler until we actually need it.